GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -1501,7 +1456,7 @@ private string GetNewProjectedShaForPlaceholder(string path)
             HashSet<string> existingPlaceholders)
         {
             bool foundFolder = this.TryGetOrAddFolderDataFromCache(relativeFolderPath, out FolderData folderData);
-            if (!foundFolder || !folderData.IsIncluded)
+            if (!foundFolder)

```

Comment:
Why was the `!folderData.IsIncluded` check removed here?

Reply:
`TryGetOrAddFolderDataFromCache` will return false when it is not included
```
if (!this.projectionFolderCache.TryGetValue(folderPath, out folderData))
{
	if (!this.TryGetFolderDataFromTreeUsingPath(folderPath, out folderData) ||
		!folderData.IsIncluded)
	{
		folderData = null;
		return false;
	}

	this.projectionFolderCache.TryAdd(folderPath, folderData);
}

return true;
```